Makefile

```diff
@@ -0,0 +1,44 @@
+############################################################################
+# This file is part of the Symfony CMF package.                            #
+#                                                                          #
+# (c) 2011-2017 Symfony CMF                                                #
+#                                                                          #
+# For the full copyright and license information, please view the LICENSE  #
+# file that was distributed with this source code.                         #
+############################################################################
+
+CONSOLE=vendor/symfony-cmf/testing/bin/console
+
+list:
+	@echo 'test:                  will run all tests'
+	@echo 'test_unit:             will run unit tests only'
+	@echo 'test_integration_orm:  will run integration tests on PHPCR'
+	@echo 'test_integration_orm:  will run integration tests on ORM'
+
+test_integration_phpcr:
+	@echo
+	@echo '+++ create PHPCR +++'
+	${CONSOLE} doctrine:phpcr:init:dbal --drop --force
+	${${CONSOLE} doctrine:phpcr:init:dbal --drop
+	${CONSOLE} doctrine:phpcr:repository:init
+	@echo '+++ run PHPCR integratin tests +++'
+	vendor/bin/simple-phpunit --testsuite "phpcr"
+	${CONSOLE} doctrine:database:drop --force
+
+test_integration_orm:
+	@echo
+	@echo '+++ Create ORM database +++'
+	${CONSOLE} doctrine:schema:drop --env=orm --force
+	${CONSOLE} doctrine:database:create --env=orm
+	${CONSOLE} doctrine:schema:create --env=orm
+	@echo '+++ run PHPCR integratin tests +++'
+	vendor/bin/simple-phpunit --testsuite "orm"
+	${CONSOLE} doctrine:database:drop --force
+
+
+test_unit:
+	@echo
+	@echo '+++ run unit tests +++'
+	vendor/bin/simple-phpunit --testsuite "Symfony SeoBundle Test Suite"
```

Comment:
should we rename this suite to "unit tests" or something like that in phpunit.xml? the current name does not indicate that its only unit tests.